samples/excel/85-preview-apis/events-formula-changed.yaml

```diff
@@ -0,0 +1,123 @@
+order: 3
+id: excel-events-formula-changed
+name: Formula change event
+description: Registers an event handler to detect changes to formulas.
+host: EXCEL
+api_set:
+    ExcelAPI: '1.13'
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup)); $("#registerFormulaChangeHandler").click(() => tryCatch(registerFormulaChangeHandler)); $("#triggerFormulaChange").click(() => tryCatch(triggerFormulaChange));
+        async function registerFormulaChangeHandler() {
+          await Excel.run(async (context) => {
+            // Retrieve the worksheet named "Sample".
+            let sheet = context.workbook.worksheets.getItem("Sample");
+            
+            // Register the formula changed event handler for this worksheet.
+            sheet.onFormulaChanged.add(formulaChangeHandler);
+            await context.sync();
+            
+            console.log("Registered a formula changed event handler for this worksheet.");
+          });
+        }
+        async function formulaChangeHandler(event: Excel.WorksheetFormulaChangedEventArgs) {
+          await Excel.run(async (context) => {
+            // Retrieve details about the formula change event.
+            const cellAddress = event.formulaDetails[0].cellAddress;
+            const previousFormula = event.formulaDetails[0].previousFormula;
+            const source = event.source;
+            
+            // Print out the change event details.
+            console.log(
+              `The formula in cell ${cellAddress} changed. 
+              The previous formula was: ${previousFormula}. 
+              The source of the change was: ${source}.`
```

Comment:
I was writing this whole thing about not including line breaks in a string literal, but apparently that's a thing you can do now in JavaScript. https://stackoverflow.com/questions/805107/creating-multiline-strings-in-javascript

So, thanks for teaching me that!

Reply:
Haha. Thanks for sharing!